### DISCUSSION TITLE
Support SNS in Contact Points

### DISCUSSION THREAD

**Discussion by jw-maynard** [grafana/grafana#43778] [Alerting]
If we were to support SNS as a contact point this would allow a way to link into Lambda.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
